corehq/apps/case_search/models.py

```diff
@@ -328,6 +330,17 @@ def case_search_synchronous_web_apps_for_domain(domain):
     return False
 
 
+@quickcache(['domain'], timeout=24 * 60 * 60, memoize_timeout=60)
+def case_search_sync_cases_on_form_entry_enabled_for_domain(domain):
```

Comment:
@stephherbers you should clear this cache when a domain saves it's case search config:
```python
case_search_sync_cases_on_form_entry_enabled_for_domain.clear(domain_name)
```

Reply:
[4a83692](https://github.com/dimagi/commcare-hq/pull/32554/commits/4a83692aec7ec7788e5a1e34769554e16e4a3fbf)